app/backend/db/crud/date_datas.py

```diff
@@ -29,11 +32,78 @@ def get_multiple_users_dates_data(
     users_id: List[int],
     start_date: date,
     end_date: date = None
-) -> List[schemas.DateResponse]:
+) -> List[schemas.RangeDatesResponse]:
+
     return [
         {
             'user_id': user_id,
             'data': get_dates_data(db, user_id, start_date, end_date)
         }
         for user_id in users_id
     ]
+
+
+def set_new_date_data(
+    db: Session, 
+    body=schemas.PostDateDataBody
```

Comment:
in crud i dont want to see parameter as body...
i prefer u to expand it here -
start_date
end_date
reason

Reply:
done